utils/entrypoint.sh

```diff
@@ -67,7 +67,7 @@ initialize () {
 
     # Look in common places for the php.ini relevant to zoneminder
     # Search order matters here because debian distros commonly have multiple php.ini's
-    for FILE in "/etc/php/7.0/apache2/php.ini" "/etc/php5/apache2/php.ini" "/etc/php.ini" "/usr/local/etc/php.ini"; do
+    for FILE in "/etc/php/7.2/apache2/php.ini" "/etc/php/7.0/apache2/php.ini" "/etc/php5/apache2/php.ini" "/etc/php.ini" "/usr/local/etc/php.ini"; do
```

Comment:
Please add the new paths to the list rather than overwrite existing paths. Overwriting is not backwards compatible.

Reply:
I don't understand this comment.

The code above adds a new path to the existing list. I have not overwritten any existing paths.

Reply:
Oh, your totally right. I misread the diff.